pom.xml

```diff
@@ -72,6 +73,16 @@
         <artifactId>opentracing-util</artifactId>
         <version>${version.io.opentracing}</version>
       </dependency>
+      <dependency>
+        <groupId>io.opentracing.contrib</groupId>
+        <artifactId>opentracing-ejb</artifactId>
+        <version>0.0.3-SNAPSHOT</version>
```

Comment:
Yeah, not merge-able while SNAPSHOT dependencies exist.
Will travis build even?